SCClassLibrary/JITLib/ProxySpace/NodeProxy.sc

```diff
@@ -896,19 +896,26 @@ NodeProxy : BusPlug {
 	// allocation
 
 	freeBus {
-		var oldBus = bus, c;
+		var oldBus = bus;
 		if(oldBus.isNil) { ^this };
-		if(this.isPlaying) {
-			c = (clock ? TempoClock.default);
-			c.sched(server.latency ? 0.01 + quant.nextTimeOnGrid(c) + this.fadeTime, { oldBus.free(true); nil });
-			CmdPeriod.doOnce { if(oldBus.index.notNil) { oldBus.free(true) } };
-		} {
-			oldBus.free(true)
-		};
+		this.schedAfterFade({
+			if(oldBus.index.notNil) { oldBus.free(true) }
+		});
 		busArg = bus = nil;
 		busLoaded = false;
 	}
 
+	schedAfterFade { |func|
+		var delta, usedClock;
+		if(this.isPlaying, {
+			usedClock = (clock ? TempoClock.default);
+			delta = server.latency ? 0.01 + this.fadeTime;
```

Comment:
magic number. why should the server latency default to 0.01?

Reply:
hm, this usually has worked best in the past. The default 0.2 is too large when you have (consciously) set latency to nil and you expect stuff to happen as fast as possible.